src/main/java/com/facebook/Application.java

```diff
@@ -1,7 +1,15 @@
 package com.facebook;
 
+import com.facebook.ui.MainUI;
+
 public class Application {
+
+    static MainUI mainUI = new MainUI();
+
     public static void main(String[] args) {
 
+        mainUI.showMainUI();
+
     }
-}
+
+}
```

Comment:
cred ca nu trebuie acolada asta de pe linia 15

Reply:
Trebuie, inchide clasa Application (incepe de la linia 5 si se termina la linia 15)

Reply:
si mie imi pare cu una in plus, cea cu rosu pare ca ar trebui sa dispara, vad ca deschidem 1 acolada la Application, una la main, dar avem 3 care se inchid

Reply:
Guys, uitati-va atent. Eu vad ca se inchide doar la linia 13 si 15, unde va ma uitati oare?
![image](https://user-images.githubusercontent.com/32415039/156388740-ae266a6a-b2f8-4458-bcd8-859a5e11698d.png)

Reply:
![image](https://user-images.githubusercontent.com/63610481/156516502-b39de4e2-c382-46ce-a40c-95498e3a94f6.png)
acuma vad ca linia pe care ne arata noua o acolada in plus nici macar nu e numerotata. Eu am crezut ca iti arata modificarile doar dupa ce ai ceva deja si faci upload la altceva dar aparent cumva noi vedem de pe-acum ca inainte o fost acolo o acolada

Reply:
Este numerotata cu 7 in prima coloana din stanga. Dar asta face partea din primul commit pus de Radu pe main branch, de aia e cu rosu.
Ce este cu verde e pus de mine pe first-feature branch, si asa se pot vedea diferentele.
La mine arata asa:
![image](https://user-images.githubusercontent.com/32415039/156565938-c5d56800-a366-4b99-ac65-0f70e7aad89f.png)

Reply:
La voi cred ca este setat Diff view pe Unified, iar la mine pe split.
Cred ca e mai ok pe split, ca sa nu apara confuzii :))
![image](https://user-images.githubusercontent.com/32415039/156566524-f168690b-020c-4334-af6b-0a12d67f2c4b.png)